content/bn/docs/reference/glossary/controller.md

```diff
@@ -0,0 +1,29 @@
+---
+title: কন্ট্রোলার
+id: controller
+date: ২০১৮-০৪-১২
+full_link: /bn/docs/concepts/architecture/controller/
```

Comment:
Does this link actually exist in [kubernetes:dev-1.24-bn.1](https://github.com/kubernetes/website/tree/dev-1.24-bn.1)? If not, we should hold this PR until that link works.